### ISSUE TITLE
Minor Bugs

### ISSUE THREAD

**Issue by Kosyne** [StreakThunderstorm/Sonic-Unreal-Worlds#6]
Can probably make separate issues out of these if desired, but just making a checklist of things I noticed so far.

- [ ] Crouching with analog has a buggy 'half state' when analog isn't all the way down.
- [x] When standing at the edge of a moving platform that pushes Sonic into a wall (swinging plats), he is teleported upwards.
- [ ] When moving up the highly angled slope at the beginning bottom, ~~you get stuck~~ (0.4 update) there's odd jittery behavior at the top angle, particularly when approached from the right. 

![reference](https://i.imgur.com/uMRuxwb.png)

**Comment by Kosyne** [comment]
Oh, didn't realize another release was around the corner. Beta 0.4 fixed one listed issue and a few I had yet to put up here, the others are still present, however.

**Comment by StreakThunderstorm** [comment]
Would you mind making a video or some gifs so I could identify the problem?

**Comment by StreakThunderstorm** [comment]
Well, not the crouch one, I was able to replicate that.

**Comment by Kosyne** [comment]
Will do. Should have em by early tomorrow.

**Comment by Kosyne** [comment]
[Sharp Angle Bug (Youtube)](https://www.youtube.com/watch?v=Aea4HFP_Ld0)

I went ahead and made one for ceiling landing as well, posting it on that issue.

**Comment by StreakThunderstorm** [comment]
Yep, this one should be fixed with angle rejection. Sonic isnt supposed to be able to walk up and down such steep angle differences.
